### ISSUE TITLE
The QR image is blurry 

### ISSUE THREAD

**Issue by wxwldu** [moqod/ios-qr-code-encoder#7]
The QR image is blurry ,how to resolve it ?

**Comment by scottrhoyt** [comment]
Perhaps you need to generate the image at a larger size?

**Comment by wxwldu** [comment]
yeah , I need a large size  on a big screen . But the large image is not a clear image.   @scottrhoyt

**Comment by Nate9210** [comment]
Wondering if there is a way to export vector for this?
